portfolio/src/main/webapp/theoffice/theoffice.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>The Office</title>
+    <link rel="stylesheet" href="theoffice.css" />
+    <link rel="stylesheet" href="/../buttons.css" />
+    <script src="theoffice.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div>
+        <a href="/../index.html" class="button button-black">Home</a>
+      </div>
+      <h1>Click here to get a random Office quote:</h1>
+      <a class="button button-purple" onclick="addRandomGreeting()">Click me!</a>
+      <h3 id="greeting-container">.</h3>
```

Comment:
remove extra dot "." in this div? Unless this is intended.

Reply:
For some reason lint complains about the empty h3, so i have to put some sort of text in there. Any ideas of how I can resolve this?

Reply:
Update, I just disabled the warning for now inside the html, there might be a way to bypass it by simply making it some other value like <p> with some styling. But its not a huge deal